keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -192,29 +192,29 @@ values in the merge patch are given special meaning to indicate the
 removal of existing values in the
 target."(link)[https://tools.ietf.org/rfc/rfc7386.txt#:~:text=Null%20values%20in%20the%20merge]
 
-On the other hand, while `{field: null}` and `{}` probably have the same
+On the other hand, while `{"field": null}` and `{}` probably have the same
 meaning, they do not "look" the same and are hard to compare for
 clients. They are also currently treated differently by Kubernetes for
 defaulting purposes.
 
-We propose the creation of a new CRD flag: `preserveNullValues` which
-will default to `true`. This flag will be used as a ratcheting mechanism
-as we migrate to this new semantic throughout all Kubernetes types
-(which would be how all types eventually work). When false, the
-apiserver will use new semantics and remove all null values from CR
-objects. When true, the existing and deprecated semantics will be kept.
-That new flag is following the same logic as "preserveUnknownFields" and
-the new operation will happen using the same mechanisms right after
-pruning.
+We propose to start removing fields with a null value, when the openapi
+nullable flag is missing or false. We assume that existing APIs that use
```

Comment:
maybe tweak the ordering to clarify explicit null values will either be removed or defaulted. saying they'll be removed here, then defaulted later is hard to follow